srcpkgs/wine/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'wine'
 pkgname=wine
 version=8.4
-revision=1
+revision=2
```

Comment:
```suggestion
revision=1
```

Reply:
Oh weird that it didn't bump the revision, thanks.

Reply:
Just read up and I gotcha, it's because there is no change to the generated binary package.